packages/main/src/Select.ts

```diff
@@ -286,10 +102,200 @@ const metadata = {
+	 * will be created inside the <code>ui5-select</code> so that it can be submitted as
+	 * part of an HTML form. Do not use this property unless you need to submit a form.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.Select.name
+	 * @public
+	 */
+	@property()
+	name!: string;
+
+	/**
+	 * Defines the value state of the component.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>None</code></li>
+	 * <li><code>Error</code></li>
+	 * <li><code>Warning</code></li>
+	 * <li><code>Success</code></li>
+	 * <li><code>Information</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.base.types.ValueState}
+	 * @defaultvalue "None"
+	 * @name sap.ui.webc.main.Select.valueState
+	 * @public
+	 */
+	@property({ type: ValueState, defaultValue: ValueState.None })
+	valueState!: ValueState;
+
+	/**
+	 * Defines whether the component is required.
+	 *
+	 * @since 1.0.0-rc.9
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @name sap.ui.webc.main.Select.required
+	 * @public
+	 */
+	@property({ type: Boolean })
+	required!: boolean;
+
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 *
+	 * @type {string}
+	 * @since 1.0.0-rc.9
+	 * @public
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.Select.accessibleName
+	 * @since 1.0.0-rc.15
+	 */
+	@property()
+	accessibleName!: string;
+
+	/**
+	 * Receives id(or many ids) of the elements that label the select.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.Select.accessibleNameRef
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	@property()
+	accessibleNameRef!: string;
+
+	/**
+	 * @private
+	 */
+	@property({ type: String, noAttribute: true })
+	_text?: string | null;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean, noAttribute: true })
+	_iconPressed!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	opened!: boolean;
+
+	/**
+	 * @type {sap.ui.webc.base.types.Integer}
+	 * @private
+	 */
+	@property({ validator: Integer, defaultValue: 0, noAttribute: true })
+	_listWidth!: number;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	focused!: boolean;
+
+	_syncedOptions: Array<Option>;
+	_selectedIndex: number;
+	_selectedIndexBeforeOpen: number;
+	_escapePressed: boolean;
+	_lastSelectedOption: Option | null;
+	_typedChars: string;
+	_typingTimeoutID?: Timeout | number;
+	responsivePopover!: ResponsivePopover;
+	selectedItem?: string | null;
+	popover?: Popover;
+	value!: string;
+
+	/**
+	 * Defines the component options.
+	 *
+	 * <br><br>
+	 * <b>Note:</b> Only one selected option is allowed.
+	 * If more than one option is defined as selected, the last one would be considered as the selected one.
+	 *
+	 * <br><br>
+	 * <b>Note:</b> Use the <code>ui5-option</code> component to define the desired options.
+	 * @type {sap.ui.webc.main.ISelectOption[]}
+	 * @slot options
+	 * @name sap.ui.webc.main.Select.prototype.options
```

Comment:
it should be sap.ui.webc.main.Select.prototype.default as you probably want to appear in the API Reference as the default slot

Reply:
I was recommended to not leave it nameless. I will set it to default and will keep the semantic 'option' slot name.